static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/dismiss-window/variant-1-article.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import type { CurrentABTest, SignInGateVariant } from '../../types';
+import { componentName } from '../../component';
+import {
+    isLoggedIn,
+    isInvalidArticleType,
+    isInvalidSection,
+    isIOS9,
+    setGatePageTargeting,
+    unsetUserDismissedGate,
+} from '../../helper';
+
+// pull in the show method from the design folder, which has the html template and and click handlers etc.
+import { designShow } from '../design/main-variant';
+
+// define the variant name here
+const variant = 'dismiss-window-variant-1-article';
+
+// method which returns a boolean determining if this variant can be shown on the current pageview
+const canShow: () => boolean = () => {
+    // clears any previous dismissal from gu.prefs.signin-gate
+    unsetUserDismissedGate({
```

Comment:
Do we want to clear this on every page, or just ignore the value in the canShowCheck? Same result either way